clients/admin-ui/src/app/store.ts

```diff
@@ -2,6 +2,7 @@ import { configureStore } from "@reduxjs/toolkit";
 import { setupListeners } from "@reduxjs/toolkit/query/react";
 import { createWrapper } from "next-redux-wrapper";
 
+import { reducer as configWizardReducer } from "~/features/config-wizard/config-wizard.slice";
```

Comment:
Starting a thread here because below you can see there are other reducers from the the `config-wizard` directory:
- [organization.slice](https://github.com/ethyca/fides/blob/8863df282c78f328765ab743e57ddb1b19ea65c3/clients/admin-ui/src/features/config-wizard/organization.slice.ts#L76)
- [system.slice.ts](https://github.com/ethyca/fides/blob/8863df282c78f328765ab743e57ddb1b19ea65c3/clients/admin-ui/src/features/system/system.slice#L88)

These states are unused, so I didn't notice them at first, but I'm guessing @LKCSmith was intending to use them to track objects shared between the wizard steps. If we go with this PR's approach, I should probably remove those unused slices*.

With this PR I'm introducing an alternative approach: the objects shared throughout the form would be grouped under `configWizard` instead of under a data-type name like `organization`/`system`.  This keeps the state organized by the feature.

As we build out this UI, I imagine we _will_ want an `organization` slice, but one that is for the common shared state used by most pages _other than_ the config wizard. Once the user has set up their org, that organization would be a common feature, and it wouldn't necessarily make sense for that info to live under the wizard.

\* Note: The organizationApi stuff is generic and should definitely stick around.

Reply:
Huh, [fixed link](https://github.com/ethyca/fides/blob/8863df282c78f328765ab743e57ddb1b19ea65c3/clients/admin-ui/src/features/system/system.slice.ts). I thought it was under config-wizard, but what I really searched for was reducer actions that were unused. But no reason to remove it here.

Reply:
oh got it. so the way I see it is that the Systems page will be its own feature, and so it makes sense to give it its own slice to keep with the "state organized by the feature" idea. But I totally see where you're coming from, that for the config wizard we don't really need a separate slice just for the state. So I'd vote for keep the slices that are resource specific, but move them out of config wizard (which system already is, and organization should move as well)